test/src/server/routes/entity/entity.js

```diff
@@ -80,3 +84,391 @@ describe('getDefaultAliasIndex', () => {
 		expect(getDefaultAliasIndex(aliasSet)).to.equal(2);
 	});
 });
+
+describe('deleteRelationships', () => {
+	const {Author, Work, Revision, WorkRevision, WorkHeader, Relationship, RelationshipSet, RelationshipType, bookshelf} = orm;
+	let author;
+	let editorJSON;
+	let relationshipTypeData;
+
+	before(async () => {
+		const editor = await createEditor();
+		editorJSON = editor.toJSON();
+
+		// Create RelationshipType Data
+		relationshipTypeData = {
+			description: 'test descryption',
+			id: 1,
+			label: 'test label',
+			linkPhrase: 'test phrase',
+			reverseLinkPhrase: 'test reverse link phrase',
+			sourceEntityType: 'Author',
+			targetEntityType: 'Work'
+		};
+		await new RelationshipType(relationshipTypeData)
+			.save(null, {method: 'insert'});
+	});
+	beforeEach(async () => {
+		author = await createAuthor();
+	});
+	after(truncateEntities);
+	it('it should return an empty array if there are no relationships to delete', async () => {
+		author = await new Author({bbid: author.get('bbid')}).fetch({withRelated: 'relationshipSet.relationships'});
+		const affectedEntities = await deleteRelationships(orm, null, author.toJSON());
+		expect(affectedEntities).to.have.length(0);
+	});
+	it('it should return an empty array if the other entity is a deleted entity (no dataId)', async () => {
+		// Create Entities
+		let work = await createWork();
+
+		// Create Relation and its data between two entites
+		const relationshipData = {
+			id: 1,
+			sourceBbid: author.toJSON().bbid,
+			targetBbid: work.toJSON().bbid,
```

Comment:
Wherever you can in this file, replacing call like `.toJSON().property` with `.get('property')` will save a bit of time, since serializing to JSON is an expensive operation, however convenient it may seem.

The only instances where toJSON is needed is to pass an entity to `deleteRelationships`.